dist/bump-release.sh

```diff
@@ -24,8 +24,8 @@ git pull --ff-only ${ORIGIN} master
 replace_version ${VERSION}
 git tag -a -m "${VERSIONTAG}" "${VERSIONTAG}"
 
-# bump ver to +git and push to origin
-replace_version "${VERSION}+git"
+# bump ver to placeholder and push to origin
+replace_version "${VERSION}+was-not-built-properly"
```

Comment:
It's fine. I wanted to get rid of the bumping logic until https://github.com/coreos/docs/issues/637#issuecomment-152661595